pkg/controller/migmigration/backup.go

```diff
@@ -163,6 +163,79 @@ func (t Task) getBackup(labels map[string]string) (*velero.Backup, error) {
 	return nil, nil
 }
 
+// returns duration of in-progress PVB
```

Comment:
nit: godoc follow `getPVBDuration returns/does this....` I know we don't follow in other places but a good practice IMO